docs/openapi.yaml

```diff
@@ -459,14 +488,16 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
-  /candidate/publication/{candidatePublicationId+}:
+  /candidate/publication/{candidatePublicationId}:
     get:
+      tags:
+        - Candidate
       description: Get NVI candidate by publication identifier
       summary: Get candidate by publication identifier
       operationId: getNviCandidateByPublicationIdentifier
       parameters:
         - in: path
-          name: candidatePublicationId+
+          name: candidatePublicationId
           description: The publication identifier the candidate, this must be URI UTF-8 encoded
           required: true
           schema:
```

Comment:
This is the only real change in the schema, as the linter insists `{candidatePublicationId+}` is not valid syntax (greedy operator). I'm not sure if this affects actual parsing through AWS services as the parameter is an URL, but in any case this endpoint is not actually in use and should be deleted I think.